css/main.css

```diff
@@ -1,5 +1,6 @@
 * {
-	font-size: 12px;
+	font-size: 14px;
```

Comment:
first of all 14px is the standard size for bootstrap and all styling for the theme are orientated to this size. and the menu had some styling issues with the altered font-size. 
and i think it looks a little bit nicer. especially on smart phones and tablets.